Compile/src/Producers/LeptonSFProducer.cc

```diff
@@ -35,15 +38,133 @@ std::string LeptonSFProducer::GetProducerId() const { return "LeptonSFProducer";
 
 void LeptonSFProducer::Init(ZJetSettings const& settings)
 {
-   
-    
+    // Get file
+    LOG(INFO) << this->GetProducerId() << ": Loading file " << m_sffile << ", Histogram "
+              << histoname;
```

Comment:
Is histoname here already set?

Reply:
This is set by the individual producers, which call the Parent Init at the end.

Reply:
okay, I wasn't sure if this producer could run standalone.
Then it is okay.